integreat_cms/api/v3/locations.py

```diff
@@ -90,6 +90,8 @@ def transform_poi_translation(poi_translation):
         "opening_hours": opening_hours,
         "location": transform_poi(poi),
         "hash": None,
+        "organization": poi.organization.name if poi.organization else None,
+        "barrier_free": poi.barrier_free or None,
```

Comment:
```suggestion
        "barrier_free": poi.barrier_free,
```
Changing `False` to `None` here doesn't make sense to me